charts/core/src/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,88 @@
+import { useEffect } from 'react';
+
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import {
+  borderRadius,
+  color,
+  InteractionState,
+  spacing,
+  Variant,
+} from '@leafygreen-ui/tokens';
+
+import { useChartContext } from '../ChartContext';
+
+import {
+  SortDirection,
+  SortKey,
+  SortOrder,
+  TooltipProps,
+} from './Tooltip.types';
+
+export function Tooltip({
+  sortDirection = SortDirection.Desc,
+  sortKey = SortKey.Value,
+  valueFormatter = value => `${value} GB`,
+}: TooltipProps) {
+  const { updateChartOptions } = useChartContext();
+  const { theme } = useDarkMode();
+
+  useEffect(() => {
+    let sortOrder: SortOrder = SortOrder.ValueDesc;
+
+    if (sortDirection === SortDirection.Asc) {
+      if (sortKey === SortKey.Name) {
+        sortOrder = SortOrder.SeriesAsc;
+      }
+      sortOrder = SortOrder.ValueAsc;
+    } else {
+      if (sortKey === SortKey.Name) {
+        sortOrder = SortOrder.SeriesDesc;
+      }
+    }
+
+    updateChartOptions({
+      tooltip: {
+        show: true,
+        backgroundColor:
+          color[theme].background[Variant.InversePrimary][
+            InteractionState.Default
+          ],
+        borderRadius: borderRadius[200],
+        borderWidth: 0,
+        confine: true,
+        enterable: false,
+        hideDelay: 0,
+        valueFormatter: valueFormatter
+          ? value => {
+              if (typeof value === 'number' || typeof value === 'string') {
+                return valueFormatter(value);
+              }
+
+              return '';
+            }
+          : undefined,
+        order: sortOrder,
+        padding: spacing[200],
+        showDelay: 0,
+        textStyle: {
+          fontFamily: 'Euclid Circular A Light, sans-serif',
+          fontSize: 12,
+          fontWeight: 'lighter',
```

Comment:
Can you explain this a bit more? Since this is only used here, and it's basically config and not real css, I'm not sure if I see the value in extracting it out.

Reply:
OH! I see what you're saying. You're totally right. Will update this in the next change

Reply:
Technically in the design the weight is `300` and we don't actually have a token for that. I just inlined it for now, but did use the font family token here!